gcp/README.md

```diff
@@ -0,0 +1,9 @@
+# AnVIL GCP Integration
+
+> NodeJS script that takes an `.avro` upload and loads the data into Google's Healthcare API
+
+## Goals
+
+[ ] Create a script that puts test data in Google Healthcare API
+[ ] Transform script to accept an `.avro` export and insert into Google Healthcare API
+[ ] Deploy script to run off bucket trigger on GCP (may require App Engine)
```

Comment:
+ [ ] Deploy IG to Google Healthcare API
+ [ ] Review service w/ Broad devops & security teams

Reply:
what does IG mean?

Reply:
`IG` ~ "Implementation Guide"
* https://github.com/ncpi-fhir/ncpi-fhir-ig  
* https://www.hl7.org/fhir/implementationguide.html

Reply:
ahh thank you. google was no help with that one